package.json

```diff
@@ -42,7 +42,7 @@
   "devDependencies": {
     "@types/react": "16.9.4",
     "@types/react-dom": "16.9.4",
-    "@types/react-redux": "7.1.7",
+    "@types/react-redux": "7.1.3",
```

Comment:
Is this the dependency you had to downgrade, as you mentioned in the meet?

Reply:
yes it is